seedbuilder/areas.ori

```diff
@@ -1665,53 +1673,68 @@ home: BashTree 521 326
 	conn: BashTreeDoorClosed
 		casual Open
 	conn: UpperGinsoRedirectArea
-		casual Bash
-		casual ChargeJump Dash DoubleJump
+		casual Bash WallJump
+		casual Bash Climb
+		casual Bash DoubleJump
+		casual Bash ChargeJump
+		casual Bash Grenade
 		casual ChargeJump DoubleJump Glide
+		standard ChargeJump Dash DoubleJump
 		standard ChargeJump Health=4
+		standard Lure Bash
```

Comment:
Does this consider the bash up inside the bash tree room? I can do it without double bash, but only by bashing the lantern up and then quickly grabbing it again, which does not seem adequate for standard.

Reply:
That bash didn't give me troubles personally

Reply:
Sounds fair. Will move this to expert.